### DISCUSSION TITLE
snes games wont save! (small update, partial solution)

### DISCUSSION THREAD

**Discussion by JuaninX** [sanni/cartreader#499] [General Chat]
Hi,
I have recorded Chinese repro without problems, but I cannot save the game, is there something I am missing? I have tried flash in 3.3v and 5v, when I check sram in 3.3v it shows errors, in 5v it doesn't, it doesn't matter if i flash in 3v or 5v i can't save the game.
![IMG_20220829_001002](https://user-images.githubusercontent.com/109401405/187121519-1648bbc9-388a-4684-a72b-edc8e41438f0.jpg)

I have this warning before flashing, what does it mean? should it be flashed in 3v?
![IMG_20220828_235257](https://user-images.githubusercontent.com/109401405/187121728-9d81503c-85d5-430e-9a15-d35f843243d4.jpg)

this error is displayed in 3v
![IMG_20220828_235806](https://user-images.githubusercontent.com/109401405/187121861-33015ae7-fee5-45a9-9952-4ceb87f0c06c.jpg)

this in 5v
![IMG_20220828_235903](https://user-images.githubusercontent.com/109401405/187121890-17fe2422-764f-4c90-92c7-ddafbb50b955.jpg)

this error when I try to put a save from an emulator
![IMG_20220828_235937](https://user-images.githubusercontent.com/109401405/187121993-c97d782f-a789-450d-9117-b53e406ab080.jpg)

**Comment by sanni** [comment]
SNES is 5V and SRAM writing in 5V works as your 4th image shows. 

Attach your emu save here maybe it has the wrong size.

**Comment by JuaninX** [comment]
I have flashed again in 5v, but I still can't save game, the test sram section everything is ok, I try to load a saved file and it doesn't work, this file is from an everdrive with the same rom.
[save.zip](https://github.com/sanni/cartreader/files/9444204/save.zip)

**Comment by sanni** [comment]
Ok the save works on my repro. Maybe try removing the wire mod again for flashing SRAM.

**Reply by JuaninX** [reply]
i try it, but wont work, maybe my repro is a faulty one.

**Reply by JuaninX** [reply]
try with another repro cartridge, a hirom, and another rom, it doesn't work, i can't save in super nintendo, and i can't load a saved file in cart reader, maybe there is some jumper? as in the repro of gameboy advance

**Comment by JuaninX** [comment]
I have done tests, I compared one of the pcb with a game that had not been deleted, and I realized that a resistor was located in C2, and the one that was recording (MX29LV160) was in C3, I changed its position and it worked, now I can save the game.
I have tried with two PCB  one with the same inscription, but it is an MX29LV320, I have tried the same thing but it has not worked, can someone tell me what type of resistor or capacitor are those located in C1; C2; C3, and what do they control?
![IMG_20220903_203142](https://user-images.githubusercontent.com/109401405/188292225-81041575-89e6-4129-bc34-ea004592552e.jpg)
in this board (MX29LV160) I changed the resistor from C3 to C2

![IMG_20220903_203147](https://user-images.githubusercontent.com/109401405/188292253-b5612065-03b4-4e9a-b174-63716ad472c7.jpg)
 this is the board  where I have seen the position of the resistance.

I need help with two pcb boards, hirom H-DKC AND ONE L-02-rst-BB, both are MX29LV320, a photo to see the position and size of the resistors.

**Reply by StanMOI** [reply]
What I see on your picture It's not a "resistor" but the "capacitor" (small SMD ceramic capacitors, size 0403).
This board has few reviews and the last of one they just changed name "C2" to "R2"
"C" means capacitor and "R" means resistor.
Here is components parts ;
C1 = 22 uF electrolytic capacitors  (to use only when using 5v -> 3v converter)
C2 (or R2) have to be = 10K ohm resistor for SUPER-CIC (pull-up resistor).
C3 = 0.1 uF ceramic capacitor for SUPER-CIC
C5 = 0.1 uF ceramic capacitor for DECODER

**Part Functions at this placement :**
Capacitor Function: Filters out high-frequency noise that can interrupt the function of the chips on the board
Electrolytic capacitors Function: Smooths out supply voltage for the board due to transients on the power supply, prevents quick changes in supply voltage when power is turned off
